### DISCUSSION TITLE
Sponsorship/FUNDING.yml

### DISCUSSION THREAD

**Discussion by timheuer** [open-telemetry/opentelemetry-dotnet#5147] [General]
Hi there, for those wanting to sponsor/donate to the OpenTelemetry project, would it be possible to enable a FUNDING.yaml on the repo? This helps folks small and large enable any sponsorship via one-time and/or larger sponsorships if possible.  Some organizations have some policies require some clarity on funding and where sponsorship goes, especially with OSS projects with multiple maintainers, some having a mix of different maintainers from larger organizations even.

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository

If GitHub Sponsors is not a desired option, even having this transparency in the README of the repo or otherwise clearly identified would help.  Thanks for consideration.

**Comment by austinlparker** [comment]
Organizations and individuals who would like to donate can do so via the LFX Crowdfunding tool at https://crowdfunding.lfx.linuxfoundation.org/projects/f1275c0e-7152-4e09-8d8b-6b14598afbc3

I'll take an item to see if we can get this documented on the website and in repositories.
